utils/schemas/main_config.json

```diff
@@ -370,6 +375,77 @@
       "examples": [
         "https://www.newrelic.com"
       ]
+    },
+    "documentation": {
+      "$id": "#/properties/documentation",
+      "type:": "array",
+      "title": "The documentation schema",
+      "description": "List of documentation links for this pack.",
+      "default": [],
+      "uniqueItems": true,
+      "examples": [
+        [
+          {
+          "name": "How to use this pack",
+          "url": "https://example.com",
+          "description": "A brief summary of what this doc entails."
+          }
+        ]
+      ],
+      "items": {
+        "$id": "#/properties/documentation/items",
+        "anyOf": [
+          {
+            "$id": "#/properties/documentation/items/anyOf/0",
+            "type": "object",
+            "title": "",
+            "description": "",
+            "default": "",
+            "examples": [
+              {
+                "name": "How to use this pack",
+                "url": "https://example.com",
+                "description": "A brief summary of what this doc entails."
+              }
+            ],
+            "required": [
+              "name", "url"
+            ],
+            "properties": {
+              "name": {
+                "$id": "#/properties/documentation/items/anyOf/0/properties/name",
+                "type": "string",
+                "title": "The name schema",
+                "description": "Name or title of the documentation resource.",
+                "default": "",
+                "examples": [
+                  "How to use this pack"
+                ]
+              },
+              "url": {
+                "$id": "#/properties/documentation/items/anyOf/0/properties/url",
+                "type": "string",
+                "title": "The url schema",
+                "description": "The URL for the documentation resource.",
+                "default": "",
+                "examples": [
+                  "https://example.com"
+                ]
+              },
+              "description": {
```

Comment:
i think john mentioned limiting this field to ~250 characters.

Reply:
I think that was summary field, description is going to stay long form